public/cover.css

```diff
@@ -0,0 +1,176 @@
+/*
+ * Globals
+ */
+
+/* Links */
+a,
+a:focus,
+a:hover {
+  color: #fff;
+}
+.nav .open>a, .nav .open>a:focus, .nav .open>a:hover {
+    background-color: transparent;
+    border-color: #337ab7;
+}
+
+/* Custom default button */
+.btn-default,
+.btn-default:hover,
+.btn-default:focus {
+  color: #333;
+  text-shadow: none; /* Prevent inheritance from `body` */
+  background-color: #fff;
+  border: 1px solid #fff;
+}
+
+
+/*
+ * Base structure
+ */
+
+html,
+body {
+  height: 100%;
+  background-color: #333;
+}
+body {
+  color: #fff;
+  text-align: center;
+  text-shadow: 0 0px 0px rgba(0,0,0,0);
+}
+h1 {
+    font-size: 50px;
```

Comment:
Prefer percentages to `px` for `font-size` (and `em` for other elements).

Also, are you sure you need to redefine these native values? What about simply increasing `body`'s base `font-size`? 🙂